File: src/main/kotlin/com/huawei/cangjie/psi/CjKeyword.java
package com.huawei.cangjie.psi;

import com.intellij.lang.ASTNode;
import org.jetbrains.annotations.NotNull;

public class CjKeyword extends CjElementImpl implements CjElement {


    public CjKeyword(@NotNull ASTNode node) {
        super(node);
    }

    @Override
    public <R, D> R accept(@NotNull CjVisitor<R, D> visitor, D data) {
         return  visitor.visitKeyword(this, data);
    }

}

File: src/main/kotlin/com/linqingying/cangjie/psi/CjKeyword.kt
package com.linqingying.cangjie.psi

import com.intellij.lang.ASTNode

class CjKeyword(node: ASTNode) : CjElementImpl(node), CjElement {
    override fun <R, D> accept(visitor: CjVisitor<R, D>, data: D?): R {
        return visitor.visitKeyword(this, data)
    }
}
